vcd_cli/user.py

```diff
@@ -143,3 +148,34 @@ def create(ctx, user_name, password, role_name, full_name, description, email,
         stdout('User \'%s\' is successfully created.' % u.get('name'), ctx)
     except Exception as e:
         stderr(e, ctx)
+
+@user.command(short_help='delete an user in current organization')
+@click.pass_context
+@click.argument('user_name',
+                metavar='<user_name>',
+                required=True)
+@click.option('-y',
+              '--yes',
+              is_flag=True,
+              callback=abort_if_false,
+              expose_value=False,
+              prompt='Are you sure you want to delete the user?')
+def delete(ctx, user_name):
+    """
+    Deletes an user from the current organization. Also deletes all assests (e.g. vApps, vms, catalogs etc.) owned by the user. However if any of those vApps are running, the command will result in error.
+
+\b
+    Examples
```

Comment:
Do we need to put the examples in the method level? From my understanding, only the examples in the class level will be parsed and showed to the users during help option.

Reply:
I tried running the following command
vcd user delete --help
and got back an empty help section. Having extra help at command level doesn't seem like a bad idea to me.

Reply:
we are aggregating all the examples at the `<object>` subcommand level and using the default help at the `<verb>` level. I think we should keep doing that for consistency

Reply:
Sure, I will make the changes.